Methylseq/methylseq.sh

```diff
@@ -154,71 +214,91 @@ if [ ! -f "$temp_path/$unmethyl_control/bismark_summary_report.txt" ]; then
     bismark2report
     bismark2summary
     #need to specify a nucleotide coverage report file in the above command!
-    rsync -vur $temp_path/ $seq_path
+    rsync -vur --exclude "main_genome" --exclude "unmethyl_genome" --exclude "hydroxymethyl_genome" --exclude "methyl_genome" $temp_path/ $seq_path
         echo "created report for unmethyl control"
 else 
         echo "bismark summary already completed for unmethyl control"
 fi
     
 if [ ! -f "$temp_path/$unmethyl_control/$hydroxymethyl_control/bismark_summary_report.txt" ]; then
     cd $temp_path/$unmethyl_control/$hydroxymethyl_control
-        echo "$temp_path/$unmethyl_control/bismark_summary_report.txt does not exist yet"
+        echo "$temp_path/$unmethyl_control/$hydroxymethyl_control/bismark_summary_report.txt does not exist yet"
     bismark2report
     bismark2summary
     #need to specify a nucleotide coverage report file in the above command! ^
-    rsync -vur $temp_path/ $seq_path
-        echo "created report for hydromethyl control"
+    rsync -vur --exclude "main_genome" --exclude "unmethyl_genome" --exclude "hydroxymethyl_genome" --exclude "methyl_genome" $temp_path/ $seq_path
+        echo "created report for hydroxymethyl control"
 else
     echo "bismark summary already completed for hydroxymethyl control"
 fi
 
-echo "report_controls complete"
+if [ ! -f "$temp_path/$unmethyl_control/$hydroxymethyl_control/$methyl_control/bismark_summary_report.txt" ]; then
+    cd $temp_path/$unmethyl_control/$hydroxymethyl_control/$methyl_control
+        echo "$temp_path/$unmethyl_control/$hydroxymethyl_control/$methyl_control/bismark_summary_report.txt does not exist yet"
+    bismark2report
```

Comment:
this needs to be called at the very end and not here. It's out of scope of this PR but needs to be run at the very end in after waiting for all the job arrays to complete and before doing the job arrays for any insert size analysis etc.

Reply:
Do this in the 'full reporting' jira ticket.